### DISCUSSION TITLE
Redirect user after login

### DISCUSSION THREAD

**Discussion by oliviercperrier** [expo/expo#26984] [Archived: Questions]
Hi,

I was wondering if there is a clean way of doing a redirect to the initial path after a successfull login use expo router

Example for web:

- User (not logged in) type `/dashboard` 
- User is sent to the login page
- User enter credentials and successfully login
- User is sent to the initial requested url `/dashboard`

Thanks

**Comment by zhangela** [comment]
Also have the same question!
